app/lib/lets-encrypt.server.ts

```diff
@@ -87,6 +91,30 @@ const { LETS_ENCRYPT_ACCOUNT_PRIVATE_KEY_PEM } = secrets ?? {};
  *  .initialize()
  *  .then((le) => le.recallOrder(orderUrl))
  *  .then((le) => logger.info(le.challengeBundles));
+ *
+ * const isChallengeSucceeded = await LetsEncrypt
+ *   .verifyChallenge({
+ *     domain: '_acme-challenge.foo.com',
+ *     key: '12345'
+ *   })
+ */
+
+/**
+ * Order
```

Comment:
I love how you included this in the docs to show what it will be like, nice idea.

Reply:
When I keep checking the same thing over and over again, I usually include it, thinking someone else might have the same problem later